DEPLOY.md

```
@@ -0,0 +1,55 @@
+* pct enter 100 # enter docker-host
+* docker exec -it summary_bot_opensource-summary-bot-1 /bin/bash # enter docker-container hosting bot
+
+
+## Deploying to Fly.io (Future use)
+This is something I don't do at present.
+
+* fly apps create summary-bot-aparine
+* fly deploy --build-arg INSTALL_DEV=true
+* fly ext sentry create
+
+Run the machine from e.g. https://fly.io/apps/summary-bot-aparine/machines/48eddeef724d08
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)